src/components/claimBTCSection/search/availability.tsx

```diff
@@ -0,0 +1,10 @@
+interface IAvailability {
+    isHidden: boolean;
+    text: string;
+    textColorString: string;
+}
+
+export const Availability = (props: IAvailability) => {
```

Comment:
To make code more readable, you can de-structure props here. 
`const Availability = ({isHidden, text, textColorString}: IAvailability)`